### PR TITLE
Add categories to org

### PR DESCRIPTION
**Pull request by rmwerner18**

### CODE REVIEW

**Comment by jd2rogers2** [comment]
lgtm so far but you can give me some test cases?
maybe using the new swagger console (localhost:3001/docs)
something like
if i hit the create org endpoint with categories: [blah] data i should succeed (see category in GET response)
if i hit the create org endpoint with categories: [not_blah] data i should fail the validator
cc: @esteban-gs

**Comment by rmwerner18** [comment]
@jd2rogers2 I haven't written tests before but gave it a shot, let me know how I did/what I need to change!

**Review by esteban-gs — APPROVED** [review]
LGTM! Should be good to go after you solve the merge conflicts

**Pull request merged**
